template.rb

```diff
@@ -117,6 +119,31 @@ def modify_menu(namespace = nil)
   end
 end
 
+def modify_rakefile
+  append_to_file "RakeFile" do
+    <<-END.strip_heredoc
+
+      namespace :sandbox do
+        desc "Reseed the database"
+        task reseed: :environment do
+          Rake::Task["sandbox:reset"].invoke
+          Rake::Task["db:schema:load"].invoke
+          Rake::Task["db:seed"].invoke
+        end
+        desc "Reset the database"
+        task reset: :environment do
+          ActiveRecord::Base.connection.tables.each do |table|
+            if table != "schema_migrations"
+              query = "DROP TABLE IF EXISTS \#{table} CASCADE;"
+              ActiveRecord::Base.connection.execute(query)
+            end
+          end
```

Comment:
Or a `Rake::Task["db:reset"] `, yes that would have been nice. But sadly Heroku, where the sandbox is hosted, does not permit either of those two (https://devcenter.heroku.com/articles/rake).